service/timeline.go

```diff
@@ -55,24 +55,11 @@ func (uis *UIServer) myPatchesTimeline(w http.ResponseWriter, r *http.Request) {
 }
 
 func (uis *UIServer) userPatchesTimeline(w http.ResponseWriter, r *http.Request) {
-	user := MustHaveUser(r)
 	author := gimlet.GetVars(r)["user_id"]
-	if user.Settings.UseSpruceOptions.PatchPage {
-		http.Redirect(w, r, fmt.Sprintf("%s/patches/user/%s", uis.Settings.Ui.UIv2Url, author), http.StatusTemporaryRedirect)
-		return
-	}
```

Comment:
Why don't we need this anymore (+ the one below)? Is it because this route is unreachable by a user?

Reply:
I think the PatchPage variable used to be how we determined if the user was opting in or out of the new patch page? (cc @khelif96  to verify)

Regardless, we don't set the variable anymore since it's deprecated